nrf52-hal-common/src/uarte.rs

```diff
@@ -72,23 +67,18 @@ impl<T> Uarte<T> where T: UarteExt {
         });
 
         // Enable UARTE instance
-        uarte.enable.write(|w|
-            w.enable().enabled()
-        );
+        uarte.enable.write(|w| w.enable().enabled());
 
         // Configure
-        uarte.config.write(|w|
-            w
-                .hwfc().bit(
-                    pins.rts.is_some() && pins.cts.is_some()
-                )
-                .parity().variant(parity)
-        );
+        uarte.config.write(|w| {
+            w.hwfc()
+                .bit(pins.rts.is_some() && pins.cts.is_some())
+                .parity()
+                .variant(parity)
```

Comment:
Call me unsophisticated, but I never understood what rustfmt is good for. And here's an example where it makes the code much less readable. `variant` clearly belongs behind `parity`, and `bit` belongs behind `hwfc`.

I'm not going to make a fuss about this, if you disagree, but in my opinion this PR would be much improved, if you dropped that last commit.

Reply:
Oops, I'm referring to the rustfmt commit (3 of 4), not the last commit. That one's good :)

Reply:
I'm in general a fan of Rustfmt, however in this case (and really, when peripheral crates in general) it tends to make poor tradeoffs.

I'll go ahead and rebase that commit out, I specifically made it it's own commit for that reason :)